src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.html

```diff
@@ -104,8 +108,8 @@ <h4>{{'general.table.header.eventTimestamp.header' | translate }}</h4>
   <div class="p-field col-12 md:col-4">
 
     <label for="delayCode">{{"general.delaycode.label" | translate}}</label>
-    <p-dropdown appendTo="body" id="delayCode" [options]="delayCodeOptions" [(ngModel)]="delayCode" [filter]="true"
-      filterBy="label,value.name" [group]="false">
+    <p-dropdown appendTo="body" id="delayCode" [options]="delayCodeOptions" [(ngModel)]="delayCode"
+      [ngModelOptions]="{standalone: true}" [filter]="true" filterBy="label,value.name" [group]="false">
```

Comment:
Would `name="foo"` avoid the need for standalone?

Reply:
using the `name` attribute on an element binds it to the parent form, this means that we have to define it in the component's formgroup.

Reply:
Bringing me to - how come we are using formgroups all of a sudden instead of sticking to template based validation?

Reply:
To add validation to the new field as well as the vessel positions. Maybe I can add them all to the form group for consistency?

Reply:
We had validations before without using form groups, so I do not understand your argument.

Reply:
We had boolean validations for whether a field is populated. 
We did **not** have pattern/custom validation rules (relevant for #201) or even show relevant warnings. This is easier done using a reactive form, which we also do in other parts of our UI.

Reply:
Custom validation I see (pattern is supported via template based).

If we are going towards form group validation, I would prefer that we converted the entire page to use it (rather than mix-match on the same page, which will be confusing in the end).

Reply:
I will convert all the fields to reactive form and rebase #201